addons/stock/models/stock_move.py

```diff
@@ -1067,6 +1067,9 @@ def _create_extra_move(self):
                         break
         return extra_move
 
+    def _unreserve_initial_demand(self, new_move):
+        return self
```

Comment:
Both methods should return the same. Either return nothing (then `pass` here) or return the result of the `write` (then `return True` here).